app/validators/questionnaire_validator.py

```diff
@@ -107,6 +108,43 @@ def validate_smart_quotes(self):
                         pointer=translatable_item.pointer,
                     )
 
+    def validate_html(self):
+        html_strings = []
+        schema_object = SurveySchema(self.schema_element)
+
+        # pylint: disable=invalid-string-quote
+        html_regex = re.compile(r"<[^>]*>")
+
+        for translatable_item in schema_object.translatable_items:
+            schema_text = translatable_item.value
+
+            values_to_check = [schema_text]
+
+            if isinstance(schema_text, dict):
+                values_to_check = schema_text.values()
+
+            for schema_text in values_to_check:
+                if schema_text and html_regex.search(schema_text):
+                    html_strings.append(
+                        {"pointer": translatable_item.pointer, "text": schema_text}
+                    )
+
+        if html_strings:
+            self.check_invalid_html_tags(html_strings)
+
+    def check_invalid_html_tags(self, html_strings):
+        strong = re.compile("<strong[^>]>*>|</strong>")
+        anchor = re.compile("<a[^>]>*>|</a>")
```

Comment:
Possibly can make it more precise by using something like: `<a .*>.*<\/a>`?

Reply:
Also handling potential multiple instances now as per strong change